src/ReactiveUI/RxApp.cs

```diff
@@ -135,7 +136,25 @@ static RxApp()
         /// </summary>
         public static IScheduler MainThreadScheduler
         {
-            get => _unitTestMainThreadScheduler ?? _mainThreadScheduler;
+            get
+            {
+                if (_unitTestMainThreadScheduler != null)
+                {
+                    return _unitTestMainThreadScheduler;
+                }
+
+                // If Scheduler is DefaultScheduler, user is likely using .NET Standard
+                if (!_hasSchedulerBeenChecked && _mainThreadScheduler == Scheduler.Default)
+                {
+                    _hasSchedulerBeenChecked = true;
+                    LogHost.Default.Warn("It seems you are running .NET Standard, but there is no host package installed!\n");
+                    LogHost.Default.Warn("You will need to install the specific host package for your platform (ReactiveUI.WPF, ReactiveUI.Blazor, ...)\n");
+                    LogHost.Default.Warn("You can install the needed package via NuGet");
```

Comment:
Looks good, but how about referencing the [installation instructions](https://reactiveui.net/docs/getting-started/installation/)? :book: 
```suggestion
                    LogHost.Default.Warn("You can install the needed package via NuGet, see https://reactiveui.net/docs/getting-started/installation/");
```

Reply:
@worldbeater you're right, a reference should be added